SPECS/tpm2-tss/tpm2-tss.spec

```diff
@@ -90,6 +69,9 @@ fi
 %{_mandir}/man7
 
 %changelog
+* Thu Feb 01 2024 Brian Fjeldstad <bfjelds@microsoft.com> - 4.0.1-3
```

Comment:
Since none of these changes have built/been published, you can keep everything at release 1 (`4.0.1-1`) if you'd like

Reply:
leaving version unchanged in PR now